src/directives/NgGrid.ts

```diff
@@ -208,6 +211,9 @@ export class NgGrid implements OnInit, DoCheck, OnDestroy {
 						this._maxCols = this._getContainerColumns();
 					}
 					break;
+				case 'allow_overlap':

```

Comment:
I think this needs to also confirm that there is no cascade setting used, i.e. cascade should be set to off. Otherwise, this is implying to the user that the grid will still cascade, despite the fact that `allow_overlap` cannot be used at the same time.